kubernetes/zenko/templates/maintenance/secrets.yaml

```diff
@@ -1,15 +1,15 @@
-{{- if .Values.maintenance.stalled.enabled -}}
+{{- if .Values.maintenance.enabled -}}
 apiVersion: v1
 kind: Secret
 metadata:
-  name: {{ template "zenko.fullname" . }}-retry-stalled-secrets
+  name: {{ template "zenko.fullname" . }}-retry-secrets
```

Comment:
We should probably template full names.

Reply:
What do you mean template full names? Isn't that what is going on here? Or you do you mean to include the "retry-secrets" ?